frontend/app/src/components/helper/display/icons/EvmChainIcon.vue

```diff
@@ -0,0 +1,53 @@
+<script setup lang="ts">
+import { type EvmChain } from '@rotki/common/lib/data';
+import { type PropType } from 'vue';
+import { getChainData } from '@/store/blockchain/chains';
+
+const props = defineProps({
```

Comment:
For new components we should prefer the `defineProps<>()` `withDefaults()` syntax of vue3 and slowly update any existing ones when we touch components